### PR TITLE
Add JUnit XML reports per Kyma integration tests

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Add JUnit XML reports per Kyma integration tests

This PRs add the kyma-testing.sh script which is using the Kyma CLI to execute Kyma integration tests and generate JUnit XML report which is consumed by Prow Job.

If this solution will be stable that we will move other jobs to also use the `kyma-testing.sh` script from test-infra instead of using `testing.sh` script from kyma repo.

Here you can find information about the next changes that are planned:
https://gist.github.com/mszostok/497fcee59651220920477260607153e3

**Related issue(s)**

- https://github.com/kyma-project/test-infra/issues/1636

### CODE REVIEW

**Comment by mszostok** [comment]
Example executions: https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/6176/pre-master-kyma-kind-lite/1192820152521986048

File: prow/scripts/lib/testing-helpers.sh [hunk 1]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
```
Comment (adamwalach) [inline prow/scripts/lib/testing-helpers.sh]: This file has different permissions than others in the same dir (its not executable).

Comment (mszostok) [reply prow/scripts/lib/testing-helpers.sh]: fixed

**Pull request merged**